src/components/events/EventForm.jsx

```diff
@@ -33,7 +35,13 @@ class EventForm extends React.Component {
 
   handleChange = (e) => {
     const { name, value } = e.target;
-    this.setState({ [name]: value });
+    console.log(value);
+    if(name === 'eventDate' && value){
+      this.setState({ [name]: value.search(/-/) === -1 ? moment(value, 'MM/DD/YYYY').format('DD/MM/YYYY') : moment(value, 'YYYY-MM-DD').format('DD/MM/YYYY') });
```

Comment:
You could segment thiis into multiple lines for readability

```
this.setState({
   [name] .....
});
```

Typically we aim for ~100 characters a line as a max line length. Anything past that and you should start playing around with how the code is formatted.